Sources/FluentKit/Properties/Parent.swift

```diff
@@ -97,7 +97,15 @@ extension ParentProperty: AnyCodableProperty {
 
     public func decode(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: ModelCodingKey.self)
-        try self.$id.decode(from: container.superDecoder(forKey: .string("id")))
+        let unwrapper = try container.decode(DecoderUnwrapper.self, forKey: .string("id"))
+        try self.$id.decode(from: unwrapper.decoder)
+    }
+}
+
+private struct DecoderUnwrapper: Decodable {
+    let decoder: Decoder
+    init(from decoder: Decoder) throws {
+        self.decoder = decoder
```

Comment:
```suggestion
        try self.$id.decode(from: container.superDecoder(forKey: .string("id")))
```
If this doesn't work, the implementation of the underlying `KeyedDecodingContainer` is wrong. Ref: https://github.com/apple/swift-corelibs-foundation/blob/main/Sources/Foundation/JSONEncoder.swift#L1604-L1618